db/version_set.cc

```diff
@@ -5278,6 +5278,14 @@ Status VersionSet::WriteCurrentStateToManifest(
       assert(iter != curr_state.end());
       uint64_t log_number = iter->second.log_number;
       edit.SetLogNumber(log_number);
+
+      if (cfd->GetID() == 0) {
```

Comment:
Why does it only apply to default CF?

Reply:
min_log_number_to_keep is for the whole db, not for specific column family.
So it does not need to be set for every column family, just need to be set once.
Since default CF can never be dropped, we set the min_log to the default CF here.

Reply:
maybe add a comment for that?